tests/006-test-overlay-cleanup

```diff
@@ -0,0 +1,39 @@
+source $SRCDIR/libtest.sh
+
+# Test "docker-storage-setup reset". Returns 0 on success and 1 on failure.
+test_reset_overlay() {
+  local testname=`basename "$0"`
+
+  # Create config file
+  clean_config_files
+  cat << EOF > /etc/sysconfig/docker-storage-setup
+STORAGE_DRIVER=overlay
+EOF
+
+ # Run docker-storage-setup
+ $DSSBIN >> $LOGS 2>&1
+
+ # Test failed.
+ if [ $? -ne 0 ]; then
+    echo "ERROR: $testname: Failed."
+    clean_config_files
+    return 1
+ fi
+
+ test_status=0
+ $DSSBIN --reset >> $LOGS 2>&1
+ if [ $? -ne 0 ]; then
+    # Test failed.
+    test_status=1
+ elif [ -e /etc/sysconfig/docker-storage ]; then
+    # Test failed.
+    echo "  $DSSBIN reset Failed: $(basename testname)"
+    test_status=1
```

Comment:
We will need to cleanup config file even in case of failure.

Also we already did "basename" on testname. So no need to do it again.